.github/pull_request_template.md

```
@@ -1,10 +1,13 @@
-NOTE: this repository uses a "Merge Forward" strategy
+## Backporting
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)